package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/cosmos",
-  "version": "2.0.3",
+  "version": "2.0.4",
```

Comment:
Did we just forget to check this in last time?

Reply:
We should add to the version checking test to also look in package-lock.json to make sure we don't forget it.

Reply:
🤦‍♂️ Seems like npm should not allow this....

I may just write our own publish script. I keep running into things like this.